Stellar-contract.x

```diff
@@ -255,6 +256,20 @@ struct Int128Parts {
     uint64 hi;
 };
 
+enum SCAddressType
+{
+    SC_ADDRESS_TYPE_CLASSIC_ACCOUNT = 0,
```

Comment:
I think we should remove the term classic here. On Stellar an account is an account.
```suggestion
    SC_ADDRESS_TYPE_ACCOUNT = 0,
```

Reply:
That makes sense. This is in artifact from the prototype where there is also 'smart' account.